### DISCUSSION TITLE
Let's ask anything, this is  solving center!

### DISCUSSION THREAD

**Discussion by superstar1205** [Web-development-s/Stripe-with-laravel#14] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prowebdev119** [comment]
Hello How are you today?

**Reply by prowebdev119** [reply]
I heard that you are ChiAnDae. Is this right?

**Comment by superstar1205** [comment, marked as answer]
Good
